lib/portlayer/util/util.go

```diff
@@ -24,6 +25,9 @@ import (
 const (
 	// XXX leaving this as http for now.  We probably want to make this unix://
 	scheme = "http://"
+
+	maxVMNameLength = 80
+	shortIDLen      = 12
```

Comment:
I think they are generic enough to deserve getting moved into lib/portlayer/constants/constants.go